libs/components/tabs/src/lib/modules/tabs/tab.component.ts

```diff
@@ -30,27 +30,27 @@ export class SkyTabComponent implements OnChanges, OnDestroy {
    * @default false
    */
   @Input()
-  public set active(value: boolean) {
-    if (value !== undefined && value !== this._active) {
-      this._active = value;
-      this._activeChange.next();
+  public set active(value: boolean | undefined) {
+    if (value !== undefined && !!value !== this.#_active) {
+      this.#_active = value;
+      this.#_activeChange.next();
     }
   }
 
   public get active(): boolean {
-    return this._active || false;
+    return this.#_active;
   }
 
   public get activeChange(): Observable<void> {
-    return this._activeChange.asObservable();
+    return this.#_activeChangeObs;
```

Comment:
This seems backwards since there's also `this.#_activeChange` property. Since `activeChange` is only referenced internally, I don't think we need the `.asObservable()` version at all and can rely on the get accessor's `Observable` type to keep other code from calling non-`Observable` methods like `next()` on it.

Reply:
I'm not sure what seemed backwards as I was following the convention that we've had for strict mode and that always relies on two properties. I got that from Steve - we can flex if needed. However, I do see what you are saying about these being internal so I changed it to only use the subject without having a field that uses `asObservable`.